deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4940,10 +4998,18 @@ NAN_METHOD(WebGLRenderingContext::GetSupportedExtensions) {
   int numExtensions = sizeof(webglExtensions)/sizeof(webglExtensions[0]);
   Local<Array> result = Nan::New<Array>(numExtensions);
   for (GLint i = 0; i < numExtensions; i++) {
-    // char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
-    result->Set(i, JS_STR(webglExtensions[i]));
+    char *extension = (char *)glGetStringi(GL_EXTENSIONS, i);
+    if(extension == webglExtensions[i]){
+      result->Set(i, JS_STR(webglExtensions[i]));
```

Comment:
This will leave holes in the output array.

Reply:
8a5cd49

Reply:
I think there will still be holes at the end of the array, since the [length is preallocated](https://github.com/exokitxr/exokit/commit/8a5cd4929f9fd348b3cdc0fa019dd6ae323e946d#diff-9830800c0783d1a603a0e5fc068f7d67R4999).

Reply:
e760558